src/components/pages/Triggers/TriggersList/TriggersList.tsx

```diff
@@ -0,0 +1,122 @@
+import {useContext, useEffect, useMemo, useState} from 'react';
+
+import {ExternalLink} from '@atoms';
+
+import {DashboardContext, MainContext} from '@contexts';
+
+import {Button, Modal, Skeleton, Text} from '@custom-antd';
+
+import {PageBlueprint} from '@organisms';
+
+import {Permissions, usePermission} from '@permissions/base';
+
+import {useGetTriggersKeyMapQuery, useGetTriggersListQuery} from '@services/triggers';
+
+import {useShallowGlobalStore} from '@store/GlobalStore';
+
+import {externalLinks} from '@utils/externalLinks';
+import {safeRefetch} from '@utils/fetchUtils';
+
+import AddTriggerModal from './AddTriggersModal';
+import EmptyTriggers from './EmptyTriggers';
+import {StyledTriggersGrid, StyledTriggersSkeletonWrapper, TriggerContainer} from './TriggersList.styled';
+
+const Triggers: React.FC = () => {
+  const {triggersList, setTriggersList, setTriggersKeyMap} = useShallowGlobalStore(state => ({
+    triggersList: state.triggersList,
+    setTriggersList: state.setTriggersList,
+    setTriggersKeyMap: state.setTriggersKeyMap,
+  }));
```

Comment:
Why do we need it? We use RTK here, and it's not used anywhere else in app

Reply:
I tried to repeat pattern as we have for other entities. I can remove it from store if you think we should not keep it just in case